lib/modules/platform/bitbucket/index.ts

```diff
@@ -945,16 +946,66 @@ export async function createPr({
           },
         )
       ).body;
-      const pr = utils.prInfo(prRes);
+      pr = utils.prInfo(prRes);
       await BitbucketPrCache.addPr(
         bitbucketHttp,
         config.repository,
         renovateUserUuid,
         pr,
       );
-      return pr;
     }
   }
+
+  logger.debug(
+    {
+      repository: config.repository,
+      title,
+      base,
+      bbAutoResolvePrTasks: platformPrOptions?.bbAutoResolvePrTasks,
+    },
+    'Auto resolve PR tasks',
+  );
+  if (platformPrOptions?.bbAutoResolvePrTasks) {
+    const listTaskRes = (
```

Comment:
Code below here to be wrapped in try/catch so that a tasks completion problem doesn't blow up the entire createPr() method